cypress/e2e/accessibility.cy.ts

```diff
@@ -1,63 +1,71 @@
 describe("Accessibility tests in light mode", () => {
     beforeEach(() => {
         cy.login();
-        cy.get(".light-button").click();
     });
 
     it("Checks clients page", () => {
         cy.visit("/clients");
 
+        cy.get(".light-button").click();
```

Comment:
Yes, because when you visit a page it is like copy-and-pasting the link and open it, rather than clicking the button on the navbar to navigate. If you try this you will realise it resets the light/dark mode